README.md

```diff
@@ -1,186 +1,248 @@
+
+It aims to remove the gap between different Heterogeneous devices, and provide a unified interface for users to manage with no changes to their applications. As of December 2024, HAMi has been widely used not only in Internet, public cloud and private cloud, but also broadly adopted in various vertical industries including finance, securities, energy, telecommunications, education, and manufacturing. More than 50 companies or institutions are not only end users but also active contributors. 
+
+![cncf_logo](imgs/cncf-logo.png)
+
+HAMi is a sandbox and [landscape](https://landscape.cncf.io/?item=orchestration-management--scheduling-orchestration--hami) project of  
+[Cloud Native Computing Foundation](https://cncf.io/)(CNCF), 
+[CNAI Landscape project](https://landscape.cncf.io/?group=cnai&item=cnai--general-orchestration--hami).
+
+
+## Device virtualization
+
+HAMi provides device virtualization for several heterogeneous devices including GPU, by supporting device sharing and device resource isolation. For the list of devices supporting device virtualization, see [supported devices](#supported-devices)
+
+### Device sharing
+
+- Allows partial device allocation by specifying device core usage.
+- Allows partial device allocation by specifying device  memory.
+- Imposes a hard limit on streaming multiprocessors.
+- Requires zero changes to existing programs.
+
+<img src="./imgs/example.png" width = "500" /> 
+
+### Device Resources Isolation
+
+A simple demonstration of device isolation:
+A task with the following resources:
+
+```
+      resources:
+        limits:
+          nvidia.com/gpu: 1 # requesting 1 vGPU
+          nvidia.com/gpumem: 3000 # Each vGPU contains 3000m device memory
+```
+
+will see 3G device memory inside container
+
+![img](./imgs/hard_limit.jpg)
+
+### Supported devices
+
+[![nvidia GPU](https://img.shields.io/badge/Nvidia-GPU-blue)](https://github.com/Project-HAMi/HAMi#preparing-your-gpu-nodes)
+[![cambricon MLU](https://img.shields.io/badge/Cambricon-Mlu-blue)](docs/cambricon-mlu-support.md)
+[![hygon DCU](https://img.shields.io/badge/Hygon-DCU-blue)](docs/hygon-dcu-support.md)
+[![iluvatar GPU](https://img.shields.io/badge/Iluvatar-GPU-blue)](docs/iluvatar-gpu-support.md)
+[![mthreads GPU](https://img.shields.io/badge/Mthreads-GPU-blue)](docs/mthreads-support.md)
+[![ascend NPU](https://img.shields.io/badge/Ascend-NPU-blue)](https://github.com/Project-HAMi/ascend-device-plugin/blob/main/README.md)
+[![metax GPU](https://img.shields.io/badge/metax-GPU-blue)](docs/metax-support.md)
+
+## Architect
+
+<img src="./imgs/hami-arch.png" width = "600" /> 
+
+HAMi consists of several components, including a unified mutatingwebhook, a unified scheduler extender, different device-plugins and different in-container virtualization technics for each heterogeneous AI devices.
+
+## Change Log

```

Comment:
I don't recommend putting changelog in readme. Changelog is a very detailed version release record. If there is a very critical breaking change, we can highlight it. Readme needs to be concise and clear so that users can quickly understand HAMi

Reply:
refer to https://github.com/hiyouga/LLaMA-Factory/tree/main and https://github.com/kubesphere/kubesphere. During this stage, we should at least show updates of important versions, so that more users can understand them clearly.